### ISSUE TITLE
Projector hangs on Computing PCA when using Tensorboard2.10 with Pytorch

### ISSUE THREAD

**Issue by XiangL-Repo** [tensorflow/tensorboard#5924]
it seems tensorboard 2.10 introduces this problem, I tried to revert to 2.9.1 version. and the hanging disappears.

**Comment by rileyajones** [comment]
Could you please provide some clear steps on how to reproduce this?

**Comment by Bomme** [comment]
I can reproduce the issue with `torch==1.12.1 tensorboard==2.10.0` by running the following code:

```python
import torch
from torch.utils.tensorboard import SummaryWriter

x = torch.rand((100, 25))

writer = SummaryWriter()
writer.add_embedding(x)
```

and then opening the projector view.

**Comment by rileyajones** [comment]
This seems to have been caused by the bundler change in #5829

**Comment by baridfor3** [comment]
Any solution? I got the same issue.

**Comment by rileyajones** [comment]
@baridfor3 We are currently recommending using 2.9.1, this is a build issue and it's hard to say how long it will take to get this patched.

**Comment by baridfor3** [comment]
@rileyajones Thanks for your answer.   2.9.1 works for me.

**Comment by bmd3k** [comment]
Thanks for bringing this to our attention. The projector plugin should now be fixed in TensorBoard 2.10.1.
